types/vexdb/README.md

```diff
@@ -0,0 +1,9 @@
+# Installation
```

Comment:
I think the `README.md` is not required here, there will be one generated when published to npm